src/components/Tree/components/EditableNodeItem.tsx

```diff
@@ -62,8 +66,13 @@ export const EditableNodeItem = ({ name, targetItemId, onEditableSave }: Editabl
                     />
                 </div>
             ) : (
-                <div data-test-id="node-link-name" onDoubleClick={handleDoubleClick}>
+                <div
+                    data-test-id="node-link-name"
+                    className="tw-flex tw-items-center"
+                    onDoubleClick={handleDoubleClick}
+                >
                     {name}
+                    {badge}
```

Comment:
Do you know if you need more than 1 badge component at a time? Otherwise, it might be then simpler to just add name and badge through the children prop. What do you think?